Automattic/jetpack
# Widgets: Hide Simple Payments create and edit buttons in widgets.php

Context: https://github.com/Automattic/wp-calypso/issues/25576

The upcoming Simple Payments widget does not work in the Appearance/Widget page.
At a first look, it may require a heavy rework of its JS logic, that could probably make us miss the next JP release.

This is not exactly a fix, but a quick (and dirty) way to hide a feature from a section where it's broken, while keeping it in the section where it works, and all by remaining on time for the next JP release.

#### Changes proposed in this Pull Request:

* In the Customizer: show the Simple Payments widget's Create and Edit buttons.
* In the Appearance/Widget page: replace them with a message linking to the Customizer's widget panel.

| Customizer | Appearance/Widgets |
| --- | --- |
| <img width="297" alt="screen shot 2018-06-22 at 11 58 13" src="https://user-images.githubusercontent.com/2070010/41773327-9c44fed4-7613-11e8-9c0b-09a48953549d.png"> | <img width="462" alt="screen shot 2018-06-25 at 15 00 19" src="https://user-images.githubusercontent.com/2070010/41854627-99cd4ae0-7888-11e8-8936-4e01d3eeb0b7.png"> |

#### Testing instructions:

* Apply this PR on a Premium/Pro site.
* Navigate to `wp-admin/customize.php`, open the Widgets panel, and add a Simple Payments widget.
* Make sure it shows the "Add New" and "Edit Selected" buttons.
* Navigate to `wp-admin/widgets.php`, and add a Simple Payments widget.
* Make sure it doesn't show those buttons, but instead a message linking to the Customizer.
* Also, make sure that the link to the Customizer actually opens its Widgets panel.
